### PR TITLE
chart merge

### PR DESCRIPTION
**Pull request by pepov**
- copy the logging-operator-logging chart from the helm-charts repo and simplify it until it works
- add a test receiver deployment to the chart and an example values file to be able to test effectively a simple scenario. my hope is that we can also use this as a quickstart
- copy docs generation from the helm-charts repo
- get rid of the charts in the e2e folder

### CODE REVIEW

File: charts/logging-operator/README.md [hunk 1]
```diff
@@ -9,8 +9,7 @@ Logging operator for Kubernetes based on Fluentd and Fluentbit.
 ## TL;DR;
 
 ```bash
-helm repo add kube-logging https://kube-logging.github.io/helm-charts
-helm install --generate-name --wait kube-logging/logging-operator
+helm install --generate-name --wait oci://ghcr.io/kube-logging/helm-charts/logging-operator --version 0.0.0
```
Comment (sagikazarmark) [inline charts/logging-operator/README.md:12]: The `--version 0.0.0` part shouldn't be necessary after the next release. If you don't specify a version, Helm should pick the latest release.

**Review by sagikazarmark — APPROVED** [review]
LGTM

Made a few minor comments

File: charts/logging-operator/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-![version: 4.2.3](https://img.shields.io/badge/version-4.2.3-informational?style=flat-square) ![type: application](https://img.shields.io/badge/type-application-informational?style=flat-square) ![app version: 4.2.2](https://img.shields.io/badge/app%20version-4.2.2-informational?style=flat-square) ![kube version: >=1.22.0-0](https://img.shields.io/badge/kube%20version->=1.22.0--0-informational?style=flat-square) [![artifact hub](https://img.shields.io/badge/artifact%20hub-logging--operator-informational?style=flat-square)](https://artifacthub.io/packages/helm/kube-logging/logging-operator)
+![version: 0.0.0](https://img.shields.io/badge/version-0.0.0-informational?style=flat-square) ![type: application](https://img.shields.io/badge/type-application-informational?style=flat-square) ![app version: latest](https://img.shields.io/badge/app%20version-latest-informational?style=flat-square) ![kube version: >=1.22.0-0](https://img.shields.io/badge/kube%20version->=1.22.0--0-informational?style=flat-square) [![artifact hub](https://img.shields.io/badge/artifact%20hub-logging--operator-informational?style=flat-square)](https://artifacthub.io/packages/helm/kube-logging/logging-operator)
```
[the remark this answers is not in the document]
Comment (pepov) [reply charts/logging-operator/README.md]: I removed for now

File: charts/logging-operator/README.md (hunk 1 above)
Comment (pepov) [reply charts/logging-operator/README.md]: I've added it as a separate example using a bash variable to show that the user should set it

**Review by ahma — APPROVED** [review]

**Pull request merged**